src/domain/space/space/space.service.authorization.ts

```
@@ -531,15 +482,24 @@ export class SpaceAuthorizationService {
+        newRules.push(rule);
         break;
+      }
       case SpacePrivacyMode.PRIVATE:
-        newRules.push(
-          this.createCredentialRuleReadAboutSpace(credentialCriteriasWithAccess)
+        const rule = this.authorizationPolicyService.createCredentialRule(
+          [AuthorizationPrivilege.READ_ABOUT],
+          credentialCriteriasWithAccess,
+          'Private spaces content is only visible to members'
         );
+        rule.cascade = false;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)